server/cri/v1/rpc_container_events.go

```diff
@@ -0,0 +1,29 @@
+package v1
+
+import (
+	pb "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+func (s *service) GetContainerEvents(req *pb.GetEventsRequest, ces pb.RuntimeService_GetContainerEventsServer) error {
+
+	if s.server.Config().EventedPLEG {
+		for containerEvent := range s.server.ContainerEventsChan {
+			if err := ces.Send(&containerEvent); err != nil {
+				return err
+			}
+		}
+	}
+
+	// Dummy code to generate a sample events stream
```

Comment:
we should remove this in the final version.

Reply:
Sure.